### PR TITLE
Apply project settings to o.e.equinox.security.tests

### PR DESCRIPTION
**Pull request by Torbjorn-Svensson**
1. Convert to LF and ensure LF at EOF:
```shell
find . -type f -print0 | \
  xargs -r0 perl -le 'for (@argv) { print if -f && -T }' | \
  xargs -rd'\n' dos2unix -e
```

2. Format java files:
```shell
eclipse -consolelog -nosplash -application org.eclipse.jdt.core.JavaCodeFormatter \
  -config .settings/org.eclipse.jdt.core.prefs . -data `mktemp -d`
```

### CODE REVIEW

**Comment by vogella** [comment]
I think a few things need changing here....

File: bundles/org.eclipse.equinox.security.tests/src/org/eclipse/equinox/internal/security/tests/storage/ManualTest.java [hunk 1]
```diff
@@ -60,33 +60,21 @@ public void testBasic() {
 			}
 
 			/*
-			 * In theory, all those ciphers support "Raw" key factory format - the SecretKeyFactory
-			 * algorithm should be really irrelevant. That said, Blowfish would not work with JavaCrypt
-			 * - so need to manually test all and exclude those that we know not supported
-			 * (or provide a roundtrip test - if it fails, exclude the value)
-		[keyFactory] PBKDF2WithHmacSHA1
```
Comment (Torbjorn-Svensson) [inline bundles/org.eclipse.equinox.security.tests/src/org/eclipse/equinox/internal/security/tests/storage/ManualTest.java:67]: And I suppose this is the other place.

@vogella Would you like this as an `<ul>`-list, `<ol>`-list or just as a `<pre>`-block?

**Comment by vogella** [comment]
@Torbjorn-Svensson can you update this one?

**Comment by vogella** [comment]
@Torbjorn-Svensson I do not see an update from you. Do you plan to finish this one?

**Comment by Torbjorn-Svensson** [comment]
> @Torbjorn-Svensson I do not see an update from you. Do you plan to finish this one?

I got slammed with other tasks the past 2 weeks. Last weekend, I started looking at why the TCKs failed in the other PRs, but never got to a point where I could actually see what test that was failing that previously was okay.
I hope I will have some more time to work on this in the next few days.

**Review by akurtakov — APPROVED** [review]

**Pull request merged**